proto/ibc/core/channel/v2/tx.proto

```diff
@@ -117,11 +117,10 @@ message MsgTimeout {
 
   option (gogoproto.goproto_getters) = false;
 
-  Packet                    packet             = 1 [(gogoproto.nullable) = false];
-  bytes                     proof_unreceived   = 2;
-  ibc.core.client.v1.Height proof_height       = 3 [(gogoproto.nullable) = false];
-  uint64                    next_sequence_recv = 4;
```

Comment:
removed this field as it was only required for Ordered channels which are not a thing now